mw-webapp/src/logic/waysTable/columns.module.scss

```diff
@@ -0,0 +1,9 @@
+.goalDescription {
+  display: -webkit-box;
+  overflow: hidden;
+  max-width: 150px;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  text-overflow: ellipsis;
+  word-wrap: break-word;
+}
```

Comment:
Why do we use webkit styles :
https://stackoverflow.com/questions/74603094/what-does-display-webkit-box-do-and-what-browsers-support-it